tests/test_controllers.py

```diff
@@ -0,0 +1,165 @@
+import numpy as np
+import pytest
+import torch as th
+
+import omnigibson as og
+import omnigibson.utils.transform_utils as T
+from omnigibson.robots import LocomotionRobot
+
+
+def test_arm_control():
```

Comment:
You should also consider taking a look at Hang's test in `test_robot_states_flatcache.py`